PowerFGT/Public/cmdb/firewall/proxy-address.ps1

```diff
@@ -0,0 +1,579 @@
+﻿#
+# Copyright 2019, Alexis La Goutte <alexis dot lagoutte at gmail dot com>
+# Copyright 2019, Benjamin Perrier <ben dot perrier at outlook dot com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+function Add-FGTFirewallProxyAddress {
+
+    <#
+        .SYNOPSIS
+        Add a FortiGate ProxyAddress
+
+        .DESCRIPTION
+        Add a FortiGate ProxyAddress (host-regex, url, category...)
+
+        .EXAMPLE
+        Add-FGTFirewallAddress -type ipmask -Name FGT -ip 192.2.0.0 -mask 255.255.255.0
```

Comment:
Wrong Example ?

Reply:
Updated all examples en tested them. Removed Category for now (pretty complex)